microsoft/testsuites/mshv/mshv_root_tests.py

```diff
@@ -33,12 +33,13 @@ class MshvHostTestSuite(TestSuite):
 
     def before_case(self, log: Logger, **kwargs: Any) -> None:
         node = kwargs["node"]
-        if not node.tools[Ls].path_exists("/dev/mshv", sudo=True):
-            raise SkippedException("This suite is for MSHV root partition only")
+        if not node.tools[KernelConfig].is_enabled("CONFIG_MSHV_DIAG"):
+            raise SkippedException("MSHV_DIAG not enabled, skip")
 
         if not node.tools[Ls].path_exists("/dev/mshv_diag", sudo=True):
-            raise SkippedException(
-                "mshv_diag module should be loaded on MSHV root parition."
+            raise LisaException(
+                "mshv_diag device should exist, when CONFIG_MSHV_DIAG is \
+                 enabled."
```

Comment:
the same, no need change it

Reply:
This is not true.
Pipeline reported the following error when I reverted this:

```
./microsoft/testsuites/mshv/mshv_root_tests.py:40:33: BLK100 Black would make changes.
./microsoft/testsuites/mshv/mshv_root_tests.py:40:89: E501 line too long (99 > 88 characters)

```